src/logger/loggerCommands.c

```diff
@@ -93,44 +95,57 @@ void ResetConfig(Serial *serial, unsigned int argc, char **argv)
 
 static void StartTerminalSession(Serial *fromSerial, Serial *toSerial, uint8_t localEcho)
 {
+        bool stay = true;
+        while (stay) {
+                /* Delay so we don't starve other tasks (like watchdog) */
+                delayMs(5);
 
-    while (1) {
-        char c = 0;
-        if (fromSerial->get_c_wait(&c, 0)) {
```

Comment:
The existing code will work perfectly if you change the 0 to some value. Right now the 0 means 'don't wait at all' . The parameter is number of ticks to wait.  Suggest you change it to:

if (fromSerial->get_c_wait(&c, msToTicks(IDLE_DELAY_MS))) {

where IDLE_DELAY_MS = 5  or whatever you want it to be.